Source/Editor/EditorCore/LibraryView.cpp

```diff
@@ -1667,5 +1739,159 @@ void LibraryView::OnCreateLibraryPress(Event* e)
+        moveText->SetText("Selected resources to move are:");
+
+        Widget* divider1 = new ContextMenuDivider(textBar, MenuUi::GutterColor);
+
+        int numberOfNamesPerLine = 5;
+        // This breaks up the display of the names of resources so that if a user has like 30 items selected, it won't extend off the screen
+        for (int i = 0; i < mResourcesToMove.Size(); i += numberOfNamesPerLine)
+        {
+            String resourcesToMove;
+            int nextNumberToStopAt = mResourcesToMove.Size() - i >= numberOfNamesPerLine ? (i + numberOfNamesPerLine) : mResourcesToMove.Size();
+
+            for (int j = i; j < nextNumberToStopAt; j++)
+            {
+                resourcesToMove = resourcesToMove + mResourcesToMove[j]->Name + ", ";
+            }
+
+            Text* resourceText = new Text(textBar, "NotoSans-Regular", 11);
+            resourceText->SetText(resourcesToMove);
+        }
+
+        Widget* divider2 = new ContextMenuDivider(textBar, MenuUi::GutterColor);
+        
+        Text* warningText1 = new Text(textBar, "NotoSans-Regular", 11);
+        warningText1->SetText("This operation cannot be undone, and may possibly break dependent assets.");
+        
+        Text* warningText2 = new Text(textBar, "NotoSans-Regular", 11); 
+        warningText2->SetText("Please be sure to double check the selected resources.");
+
+
+    }
+
+    Composite* buttonBar = new Composite(this);
+
+    buttonBar->SetLayout(CreateStackLayout(LayoutDirection::LeftToRight, Vec2::cZero, Thickness::cZero));
+    {
+        Composite* space = new Composite(buttonBar);
+        space->SetSizing(SizeAxis::X, SizePolicy::Flex, 1);
+
+        TextButton* createButton = new  TextButton(buttonBar);
+        createButton->SetText("Move");
+        createButton->SetSizing(SizeAxis::X, SizePolicy::Fixed, Pixels(80));
+
+        ConnectThisTo(createButton, Events::ButtonPressed, OnMove);
+
+        TextButton* cancelButton = new  TextButton(buttonBar);
+        cancelButton->SetText("Cancel");
+        cancelButton->SetSizing(SizeAxis::X, SizePolicy::Fixed, Pixels(80));
+        ConnectThisTo(cancelButton, Events::ButtonPressed, OnCancel);
+    }
+}
+
+MoveItemUI::~MoveItemUI()
+{
+}
+
+void MoveItemUI::BuildContentLibraryList()
+{
+    mContentLibraries->ClearItems();
+    mLibrariesRow->SetActive(false);
+
+    forRange(ContentLibrary * library, PL::gContentSystem->Libraries.Values())
+    {
+        if (library == nullptr || library->GetReadOnly())
+            continue;
+
+        // Only show the library if a resource library was built from it and if it's writeable
+        if (PL::gResources->GetResourceLibrary(library->Name))
+            mContentLibraries->AddItem(library->Name);
+
+    }
+
+    // Enable dropdown selection menu for multiple libraries
+    if (mContentLibraries->GetCount() > 1)
+    {
+        mLibrariesRow->SetActive(true);
+        mContentLibraries->SetSelectedItem(0, false);
```

Comment:
I doubt it matters since we are creating a new UI for each move operation, but this `SetSelectedItem` call should probably happen even in the case where `mContentLibraries->GetCount() == 1`. Not sure about the case there are 0 items in mContentLibraries.

Reply:
Ah yeah, good catch. I should change the condition to >= 1 instead.

Reply:
Nevermind that, i had to see the full block, will change it up.